### PR TITLE
PR-01

### PR DESCRIPTION
**Pull request by mrganeshraja**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

**Test Configuration**:

* Hardware:
* O/S and version:
* JDK:

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/integrationTest/resources/application-aat.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+auth.idam.client.baseUrl=http://betaDevBccidamAppLB.reform.hmcts.net
+idam.s2s-auth.url=http://betaDevBccidamS2SLB.reform.hmcts.net:80
+
+idam.auth.secret=RJBNLKOV7OJR52VX
```
Comment (mthava) [inline src/integrationTest/resources/application-aat.properties]: remove this and use the value coming from vault

File: src/integrationTest/resources/application.properties [hunk 2]
```diff
@@ -0,0 +1,20 @@
+logging.level.org.springframework.web=DEBUG
+logging.level.uk.gov.hmcts.ccd=DEBUG
+
+env=${TEST_ENVIRONMENT:local}
+
+#auth.idam.client.baseUrl=http://betaDevBccidamAppLB.reform.hmcts.net
```
Comment (mthava) [inline src/integrationTest/resources/application.properties]: Remove comment here

File: src/integrationTest/resources/application.properties [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+auth.idam.client.baseUrl=http://localhost:4501
+#idam.s2s-auth.url=http://betaDevBccidamS2SLB.reform.hmcts.net:80
```
Comment (mthava) [inline src/integrationTest/resources/application.properties]: Remove comment here

File: src/integrationTest/resources/application.properties [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+idam.auth.secret=AAAAAAAAAAAAAAAA
+idam.auth.microservice=divorce_document_generator
+
+evidence.management.client.api.baseUrl=http://localhost:4006/emclientapi/version/1
+evidence.management.client.api.endpoint.upload=/uploadFiles
+evidence.management.client.api.endpoint.download=/downloadFile
+evidence.management.client.api.endpoint.uploadwiths2stoken=/uploadFilesWithS2SAuthToken
+
+
+
```
Comment (mthava) [inline src/integrationTest/resources/application.properties]: remove extra lines

File: src/main/java/uk/gov/hmcts/reform/emclient/application/EvidenceManagementClientApplication.java [hunk 5]
```diff
@@ -1,20 +1,44 @@
...
 import org.springframework.retry.annotation.EnableRetry;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGenerator;
+import uk.gov.hmcts.reform.authorisation.generators.AuthTokenGeneratorFactory;
 
 @SpringBootApplication
 @ComponentScan(basePackages = "uk.gov.hmcts")
 @EnableAutoConfiguration(exclude = HypermediaAutoConfiguration.class)
 @EnableRetry(proxyTargetClass=true)
+//@ImportAutoConfiguration({RibbonAutoConfiguration.class,HttpMessageConvertersAutoConfiguration.class, FeignRibbonClientAutoConfiguration.class, FeignAutoConfiguration.class})
```
Comment (mthava) [inline src/main/java/uk/gov/hmcts/reform/emclient/application/EvidenceManagementClientApplication.java]: Remove this line

File: src/main/resources/application.properties [hunk 6]
```diff
@@ -13,19 +13,29 @@ spring.http.multipart.max-request-size=30000KB
 ####Evidence management Service URL's
 evidence.management.upload.file.url=${EVIDENCE_MANAGEMENT_UPLOAD_FILE_URL:http://localhost:3404/documents}
 
-evidence.management.store.upload.file.url=${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:3405}/documents
+evidence.management.store.upload.file.url=${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:4603}/documents
 
 evidence.management.store.health.url=${DOCUMENT_MANAGEMENT_STORE_URL:http://localhost:3405}/health
+#evidence.management.store.health.url="https://dm-store-app-demo.service.core-compute-demo.internal/health"
```
Comment (mthava) [inline src/main/resources/application.properties]: remove this

File: build.gradle [hunk 7]
```diff
@@ -34,6 +36,24 @@ jar {
     }
 }
 
+
```
[the remark this answers is not in the document]
Comment (m4854217199349125) [reply build.gradle]: Done alredy

File: Jenkinsfile_CNP [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+@Library("Infrastructure")
+
+def type = "java"
+def product = "div"
+def component = "em-client-api"
+
+def channel = '#div-dev'
+
+List<LinkedHashMap<String, Object>> secrets = [
+        secret("secret/test/ccidam/service-auth-provider/api/microservice-keys/divorceDocumentUpload", 'DIVORCE_DOCUMENT_UPLOAD_KEY'),
```
[the remark this answers is not in the document]
Comment (m4854217199349125) [reply Jenkinsfile_CNP]: done

File: src/integrationTest/resources/application.properties (hunk 2 above)
Comment (m4854217199349125) [reply src/integrationTest/resources/application.properties]: done

File: src/integrationTest/resources/application.properties (hunk 3 above)
Comment (m4854217199349125) [reply src/integrationTest/resources/application.properties]: done

File: src/integrationTest/resources/application.properties (hunk 4 above)
Comment (m4854217199349125) [reply src/integrationTest/resources/application.properties]: done

File: src/main/java/uk/gov/hmcts/reform/emclient/application/EvidenceManagementClientApplication.java (hunk 5 above)
Comment (m4854217199349125) [reply src/main/java/uk/gov/hmcts/reform/emclient/application/EvidenceManagementClientApplication.java]: done

File: src/main/resources/application.properties (hunk 6 above)
Comment (m4854217199349125) [reply src/main/resources/application.properties]: done

**Pull request merged**